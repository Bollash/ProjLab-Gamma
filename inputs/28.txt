Create Space
Add Asteroid Iron
Add Ufo 0
Act 0
Status All