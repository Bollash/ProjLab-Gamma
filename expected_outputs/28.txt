Space létrehozva
A nyersanyaggal teli aszteroida hozzá lett adva a space-hez.
Sikeres volt az ufóhozzáadás
Az actor nem tud szomszédos aszteroidára mozogni.
Space
0
10
10
False
1
1
Asteroid
0
3
3
3
Iron
0
1
0
Ufo
0
0
